### PR TITLE
scte35 documentation

### PR DESCRIPTION
**Pull request by rbouqueau**
waiting for https://github.com/gpac/gpac/pull/2748 to be merged

### CODE REVIEW

File: docs/Howtos/scte35-markers.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Overview
```
Comment (nlsdvl) [inline docs/Howtos/scte35-markers.md:1]: please prefer h2 for page content

File: docs/Howtos/scte35-markers.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+gpac -i input -o output
+```
+
+The format is selected automatically depending on the file extension (or can be forced with ```:ext=```).
+
+# Segmentation and 23001-18 ISOBMFF Event Track
+
+Adaptive streaming with DASH or HLS requires the content to be segmented. Simple segmentation (where content can be replicated over several segments without any change) works for all kind of streams.
+
+For SCTE-35 streams the [dec_scte35 filter](dec_scte35) must be explicitly loaded.
```
Comment (nlsdvl) [inline docs/Howtos/scte35-markers.md:87]: @rbouqueau you need to generate this file by running `gpac -genmd` and also add to the filter's documentation page to the nav structure manually

File: docs/Howtos/scte35-markers.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+Note that it can detect the segment duration from the [dasher](dasher) to create the right segmentation for you):
```
Comment (nlsdvl) [inline docs/Howtos/scte35-markers.md:88]: typo, you have an extra closing parenthesis ")"

**Comment by nlsdvl** [comment]
@rbouqueau please check my comments. 
The PR is also missing the new filter documentation. 
The new pages must be manually added to the navigation tree : https://github.com/gpac/wiki/blob/master/mkdocs.yml#L70

**Comment by rbouqueau** [comment]
> The PR is also missing the new filter documentation.

Added via updating master

> The new pages must be manually added to the navigation tree : https://github.com/gpac/wiki/blob/master/mkdocs.yml#L70

Done

**Comment by nlsdvl** [comment]
@rbouqueau the new filter `dec_scte35` and the output of `gpac -genmd` are not automatically added. Such updates need to be pushed explicitly to the wiki. You want to add that filter's page to the navigation tree as well.

**Comment by rbouqueau** [comment]
@nlsdvl Thanks. I did push it 22b8e11b490289c5ec1b8a9c6a261d1879d9acc2 but in master to avoid polluting this branch. The main PR (https://github.com/gpac/gpac/pull/2748) was merged in between.

**Comment by rbouqueau** [comment]
Can we merge?

**Comment by nlsdvl** [comment]
You did not add Filters/scte35dec.md to the index. I'll take care of it.

File: docs/Howtos/scte35-markers.md (hunk 1 above)
Comment (rbouqueau) [reply docs/Howtos/scte35-markers.md:1]: so should we also convert the other pages as well with ```##``` ?

Comment (nlsdvl) [reply docs/Howtos/scte35-markers.md:1]: I ending up writing a hook that fixes the headings on the fly: https://github.com/gpac/wiki/blob/master/scripts/mkdocs_hooks.py#L178

as long as it works, no need to fix them.

**Pull request merged**